Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -416,58 +415,58 @@ TimeNumberCombinedWithUnit: !nestedRegex
   references: [ TimeUnitRegex ]
 BusinessHourSplitStrings: ['business', 'hour']
 NowRegex: !simpleRegex
-  def: \b(?<now>(right\s+)?now|as soon as possible|asap|recently|previously)\b
+  def: \b(?<now>nu|zo snel mogelijk|zo spoedig mogelijk|asap|recent|onlangs|zojuist)\b
 SuffixRegex: !simpleRegex
-  def: ^\s*(in the\s+)?(morning|afternoon|evening|night)\b
+  def: ^\s*(in de\s+)?(vroege\s+|late\s+)?(ochtend|(na)?middag|avond|nacht)\b
 DateTimeTimeOfDayRegex: !simpleRegex
-  def: \b(?<timeOfDay>morning|afternoon|night|evening)\b
+  def: \b(?<timeOfDay>ochtend|(na)?middag|avond|nacht)\b
 DateTimeSpecificTimeOfDayRegex: !nestedRegex
-  def: \b(({RelativeRegex}\s+{DateTimeTimeOfDayRegex})\b|\btonight)\b
+  def: \b(({RelativeRegex}\s+{DateTimeTimeOfDayRegex})\b|\bvannacht|vanavond|vanmiddag|vanochtend)\b
```

Comment:
Could you add `vanmorgen`, since this is commonly used. (low prio)

Reply:
will be added.

Reply:
Resolved.